### PR TITLE
Add LAYOUT_75_iso community layout support

### PR DESCRIPTION
**Pull request by zvecr**
## Description

To go along side with LAYOUT_75_ansi, i have added a corresponding iso layout.

This is compatible with the KBD75 (which rev2 seems to be broken in the Configurator), i will add support in another PR.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
I'd prefer if it were set up so `KC_ENT` was on the home row (between `KC_NUHS` and `KC_PGDN` (makes it more straightforward to convert keymaps between ANSI and ISO, as well as debugging issues between the two layouts on the same board), but otherwise :+1:.

Trade-off is that making that change makes Configurator support for `LAYOUT_75_iso` a bit more involved.

**Comment by noroadsleft** [comment]
As an aside, KBD75 rev2 isn't broken per se; rev2 supports all the same layouts that rev1 does *plus* the LAYOUT_numpad layout shown on Configurator.

You could do a layout on `kbdfans/kbd75/rev1` and flash the .hex to a rev2, and have zero problems.

**Comment by zvecr** [comment]
@noroadsleft while a valid workaround, its not the most intuitive system, as well as not a practice that can be universally applied. For example, if the hardware is vastly different, could lead to some soft bricked boards.

**Comment by noroadsleft** [comment]
KBD75 rev2 uses the same switch matrix as rev1 for the layouts that are supported on both.

https://github.com/qmk/qmk_firmware/pull/3261

**Comment by zvecr** [comment]
I agree that the layouts are compatible, its more of an issue with presentation in the Configurator. Some other boards handle it better, like clueboard/66. I think its mainly due to the rev2.h including rev1.h. Im guessing it can be fixed in a variety of ways, duplicating macros between revisions, extending functionality within the Configurator, etc.

**Comment by zvecr** [comment]
Looking at the other default iso layouts, they all have `KC_ENT` next to `KC_NUHS`, i will update to match.

File: layouts/default/75_ansi/default_75_ansi/keymap.c [hunk 1]
```diff
@@ -8,5 +8,6 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
         KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
         KC_CAPS, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT, KC_ENT,  KC_PGDN,
         KC_LSFT, KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT, KC_SLSH, KC_RSFT,          KC_UP,   KC_END,
-        KC_LCTL, KC_LGUI, KC_LALT,                KC_SPC,                                  KC_RALT, KC_RGUI, KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT)
```
Comment (noroadsleft) [inline layouts/default/75_ansi/default_75_ansi/keymap.c]: Not sure why this file is on this PR...

Comment (zvecr) [reply layouts/default/75_ansi/default_75_ansi/keymap.c]: Shoot, i aligned the whitespace at the end of the file, must had done both iso and ansi. Shall i back it out?

Comment (noroadsleft) [reply layouts/default/75_ansi/default_75_ansi/keymap.c]: Yeah, please revert for this file. For 75_iso it's fine.

Comment (zvecr) [reply layouts/default/75_ansi/default_75_ansi/keymap.c]: Done

**Pull request merged**